src/core/tsi/ssl_transport_security_utils.h

```diff
@@ -142,6 +142,23 @@ tsi_result SslProtectorUnprotect(const unsigned char* protected_frames_bytes,
                                  unsigned char* unprotected_bytes,
                                  size_t* unprotected_bytes_size);
 
+// Verifies the CRL signature using X509_CRL_verify
```

Comment:
My 2 cents: The API description should describe what the API does, rather than leaning on explaining the implementation details.

Similarly below.

Reply:
Not addressing this yet, so please leave open. Will make the text better

Reply:
Ok this should be addressed